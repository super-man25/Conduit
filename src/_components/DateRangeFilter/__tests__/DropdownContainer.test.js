import React from 'react';
import renderer from 'react-test-renderer';
import { DropdownContainer } from '../DropdownContainer';

it('renders correctly', () => {
  const mockClickawayFunction = jest.fn();
  const tree = renderer
    .create(<DropdownContainer onClickAway={mockClickawayFunction} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
