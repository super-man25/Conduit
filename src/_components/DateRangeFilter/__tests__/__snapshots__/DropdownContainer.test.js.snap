// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  position: relative;
  cursor: pointer;
}

<div>
  <div
    className="c0"
  />
</div>
`;
